README.md

```diff
@@ -123,6 +124,32 @@ and DLT-META [documentation](https://databrickslabs.github.io/dlt-meta/)
 
 12. Make sure job run successfully. Verify metadata in your dataflow spec tables entered in step: 9 e.g ```dlt_demo.bronze_dataflowspec_table``` , ```dlt_demo.silver_dataflowspec_table```
 
+### Option#2. Notebook
```

Comment:
should the info here match with the steps/comments in the runonboardingopt2.md file?